Description: Fix LC_ALL to C in tests
 Fix LC_ALL to C for the error messages in the statement handle tests
Author: Damyan Ivanov <dmn@debian.org>
Reviewed-by: Xavier Guimard <x.guimard@free.fr>
Last-Update: 2013-05-03
Forwarded: not-needed
--- a/t/06attrs.t
+++ b/t/06attrs.t
@@ -3,6 +3,7 @@
 use strict;
 
 use Test::More;
+use POSIX qw(setlocale LC_ALL);
 
 ## ----------------------------------------------------------------------------
 ## 06attrs.t - ...
@@ -17,6 +18,9 @@ BEGIN {
 
 $|=1;
 
+# For the error messages in the statement handle tests.
+setlocale(LC_ALL, "C");
+
 my $using_autoproxy = ($ENV{DBI_AUTOPROXY});
 my $dsn = 'dbi:ExampleP:dummy';
 
